### DISCUSSION TITLE
Flagging --as-cran in version 2

### DISCUSSION THREAD

**Discussion by rkillick** [r-hub/rhub#632] [Q&A] [answered]
In v1 there was a `check_for_cran()` function that allowed us to pass flags to the `R CMD BUILD` step.  A common one to use was `--as-cran`.  In the documentation I have found, adding flags isn't covered in the new workflow (apologies if I have missed something!).  Is there a way to add flags to the `R CMD BUILD` from within R or do we have to modify the action yaml?

**Comment by gaborcsardi** [comment, marked as answer]
You cannot change the flags currently. But AFAIR `R CMD build` does not have an `--as-cran` flag. `R CMD check` does, but I think `--as-cran` is already the default there.

**Reply by rkillick** [reply]
Thanks for the quick reply.  Apologies, I meant R CMD check!

Looking at the yaml, I can't see a as-cran argument, similarly I don't see it as an option or setting in `rhub_check()`. 

I looked at the checks on Ubuntu [here](https://github.com/rkillick/PenalizedCorr/actions/runs/10306862678/job/28530871751) and compared with my local install with and without --as-cran.  There are sufficient differences between them that it isn't clear to me whether --as-cran is default or not.

**Reply by gaborcsardi** [reply]
```
* using options ‘--no-manual --as-cran’
```
https://github.com/rkillick/PenalizedCorr/actions/runs/10306862678/job/28530871751#step:6:68

**Reply by rkillick** [reply]
Ah yes, there it is.  Thank you!
